docs/get-started/getting-started.md

```diff
@@ -2,7 +2,7 @@
 
 ## Installation
 
-First, you need to install the deck.gl (and luma.gl) frameworks
+To install the deck.gl framework:
 
 ```bash
 npm install deck.gl --save
```

Comment:
Should we align this with the submodule system below?

Reply:
As a quick start this is the fastest and safest way to get things going. Advanced users who want to cherry pick their dependencies can read the section below.